src/pages/gallery/questions/index.tsx

```diff
@@ -1,31 +1,73 @@
 import React from "react";
 import classnames from "classnames";
-import { GetStaticProps } from "next";
 import Link from "next/link";
 import Head from "next/head";
-import Image from "next/image";
+import Image, { ImageProps } from "next/image";
 
 import { Heading } from "../../../components/Heading";
 import { PageBanner } from "../../../components/Banner";
 import { BannerCTA } from "../../../components/CallToActionBanner";
 import { LinkButton } from "../../../components/LinkButton";
-import { getQuestions } from "../../../lib/gallery/questions";
 
 import styles from "./index.module.scss";
 import questionImage from "../../../lib/images/question.png";
 
-interface Question {
-  title: string;
-  slug: string;
-  summary: string;
-  imageUrl?: string;
+{
+  /* Template Question Images */
 }
+import selectBird from "../../../lib/images/questionTemplates/selectBird.png";
+import selectRandomNumber from "../../../lib/images/questionTemplates/selectRandomNumber.png";
+import selectRainbowColor from "../../../lib/images/questionTemplates/selectRainbowColor.png";
+import selectPlanet from "../../../lib/images/questionTemplates/selectPlanet.png";
+import selectTrueFalse from "../../../lib/images/questionTemplates/selectTrueFalse.png";
+import selectEven from "../../../lib/images/questionTemplates/selectEven.png";
+import randomEquation from "../../../lib/images/questionTemplates/randomEquation.png";
+import fixedEquation from "../../../lib/images/questionTemplates/fixedEquation.png";
+import convertAngle from "../../../lib/images/questionTemplates/convertAngle.png";
+import dynamicImage from "../../../lib/images/questionTemplates/dynamicImage.png";
+import graphMatrix from "../../../lib/images/questionTemplates/graphMatrix.png";
+import productMatrices from "../../../lib/images/questionTemplates/productMatrices.png";
+import derivativeSymbolic from "../../../lib/images/questionTemplates/derivativeSymbolic.png";
+import inputText from "../../../lib/images/questionTemplates/inputText.png";
 
-interface GalleryIndexProps {
-  questions: Question[];
+interface TemplateCardProps {
+  title: string;
+  href: string;
+  image: ImageProps["src"];
+  children: React.ReactNode;
 }
 
-const GalleryIndex: React.FC<GalleryIndexProps> = ({ questions }) => {
+const TemplateCard: React.FC<TemplateCardProps> = ({
+  title,
+  href,
+  image,
+  children,
+}) => {
+  return (
+    <article className="card border-secondary overflow-hidden">
+      <Link href={href} className="position-relative pt-2">
+        <Image
```

Comment:
I wonder whether it would look better with a frame around the image?

Reply:
I can try, but as of now, the image covers the entire extension of the card from left to right, meaning that when I add borders to the image, I only get two horizontal lines on top and bottom of the image. it does not look good. I would prefer to avoid having to get new images, so hopefully we can come up with style improvements that can be done in the code with the current images.